### PR TITLE
fix/replace simple quote by two dollar symbol

### PR DESCRIPTION
**Pull request by gpoujol**

### CODE REVIEW

**Comment by bobeal** [comment]
I don't get why you are doing the dollar-quote escaping for *every* column, even ones we know for sure they are not concerned.

**Comment by gpoujol** [comment]
> I don't get why you are doing the dollar-quote escaping for _every_ column, even ones we know for sure they are not concerned.

Is not it better to apply the same thing everywhere than to make an exception?

**Comment by bobeal** [comment]
> > I don't get why you are doing the dollar-quote escaping for _every_ column, even ones we know for sure they are not concerned.
> 
> Is not it better to apply the same thing everywhere than to make an exception?

Everywhere where it can happen, yes. What's the point, for instance, of doing it for an UUID or a date that has already been parsed and validated?

**Comment by gpoujol** [comment]
> > > I don't get why you are doing the dollar-quote escaping for _every_ column, even ones we know for sure they are not concerned.
> > 
> > 
> > Is not it better to apply the same thing everywhere than to make an exception?
> 
> Everywhere where it can happen, yes. What's the point, for instance, of doing it for an UUID or a date that has already been parsed and validated?

Ok I will change that

File: search-service/src/main/kotlin/db/migration/V0_28__JsonLd_migration.kt [hunk 1]
```diff
@@ -194,7 +194,7 @@ class V0_28__JsonLd_migration : BaseJavaMigration() {
                             ${temporalAttributesMetadata.value.measuredValue}, 
                             ${temporalAttributesMetadata.value.value.toSQLValue()}, 
                             ST_GeomFromText('${temporalAttributesMetadata.value.geoValue!!.value}'), 
-                            '$teaId', '$attributeInstanceId', '$serializedAttributePayload')
+                            '$teaId', $$$attributeInstanceId$$, '$serializedAttributePayload')
```
Comment (bobeal) [inline search-service/src/main/kotlin/db/migration/V0_28__JsonLd_migration.kt:197]: `attributeInstanceId` is generated by us and does not need to be dollar-quoted. But `serializedAttributePayload` should.

File: search-service/src/main/kotlin/db/migration/V0_28__JsonLd_migration.kt [hunk 2]
```diff
@@ -205,7 +205,7 @@ class V0_28__JsonLd_migration : BaseJavaMigration() {
                         ('$createdAt', '${AttributeInstance.TemporalProperty.CREATED_AT}', 
                             ${temporalAttributesMetadata.value.measuredValue}, 
                             ${temporalAttributesMetadata.value.value.toSQLValue()}, 
-                            '$teaId', '$attributeInstanceId', '$serializedAttributePayload')
+                            '$teaId', $$$attributeInstanceId$$, $$$serializedAttributePayload$$)
```
Comment (bobeal) [inline search-service/src/main/kotlin/db/migration/V0_28__JsonLd_migration.kt:208]: same for `attributeInstanceId`

**Review by bobeal — APPROVED** [review]

**Pull request merged**